src/pages/_app.tsx

```diff
@@ -117,7 +117,6 @@ const CoreApp: Omit<NextAppComponentType, 'origGetInitialProps'> = ({
             <InteractionProvider>
               <ToastProvider components={{ Toast }}>
                 <Head>
-                  <title>Overseerr</title>
```

Comment:
This is the default fallback. Why did we get rid of it?

Reply:
Ah whoops, I removed it initially to make it easier for me to test something, but I'll add that back.